drivers/sensor/ti/tmag5273/tmag5273.c

```diff
@@ -102,17 +102,17 @@ static int tmag5273_reset_device_status(const struct device *dev)
 /**
  * @brief checks for DIAG_FAIL errors and reads out the DEVICE_STATUS register if necessary
  *
- * Prints a human readable representation to the log, if \c CONFIG_LOG is activated.
- *
- * @param drv_cfg[in] driver instance configuration
- * @param device_status[out] DEVICE_STATUS register if DIAG_FAIL is set
+ * @param[in] drv_cfg driver instance configuration
+ * @param[out] device_status DEVICE_STATUS register if DIAG_FAIL is set
```

Comment:
This and some other changes appear unrelated to the commit message `drivers: tmag5273: remove unneccessary check`. Please revert the unrelated changes or move them to separate commits.

Reply:
Better now?